### PR TITLE
Use golang 1.15 for compliance-operator CI builds

### PR DESCRIPTION
**Pull request by JAORMX**

### CODE REVIEW

File: ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml [hunk 1]
```diff
@@ -7,18 +7,22 @@ base_images:
     name: builder
     namespace: ocp
     tag: golang-1.14
+  ocp_builder_golang-1.15:
```
Comment (jhrozek) [inline ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml:10]: Why not add another builder and not drop the 1.14 builder above? It doesn't seem to be used anywhere anymore...

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now f8be3e8504038dd0769a088a52053ea8ce416c85

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 2c4ba2d201d55184ae740131de7be108425537bb

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now dd711c5f80d1d4f4d119df99151ce43e0265fdce

**Pull request closed without merging**